### PR TITLE
release v1.7.10

### PR DESCRIPTION
**Pull request by vankichi**
SSIA

### CODE REVIEW

File: content/docs/tutorial/get-started.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "Get Started_tutorial"
-date: 2023-12-06T16:56:22+09:00
-description: The tutorial of running Vald cluster on the Kubernetes cluster.
```
Comment (kmrmt) [inline content/docs/tutorial/get-started.md:4]: Why is this description deleted?

File: content/docs/user-guides/deployment.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "Deployment_user Guides"
-date: 2023-01-24T16:35:13+09:00
-description: Introducing how to deploy the Vald cluster.
```
Comment (kmrmt) [inline content/docs/user-guides/deployment.md:4]: Why is this description deleted?

**Review by ykadowak — APPROVED** [review]

File: content/docs/user-guides/deployment.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
...
+date: 2023-12-25T19:23:03+09:00
```
Comment (vankichi) [inline content/docs/user-guides/deployment.md:3]:
```suggestion
date: 2023-12-25T19:23:03+09:00
description: Introducing how to deploy the Vald cluster.
```

File: content/docs/tutorial/get-started.md [hunk 4]
```diff
@@ -1,7 +1,6 @@
...
+date: 2023-12-25T19:23:01+09:00
```
Comment (vankichi) [inline content/docs/tutorial/get-started.md:3]:
```suggestion
date: 2023-12-25T19:23:01+09:00
description: The tutorial of running Vald cluster on the Kubernetes cluster.
```

**Review by hlts2 — APPROVED** [review]
LGTM!!

**Review by kmrmt — APPROVED** [review]

**Pull request merged**